### ISSUE TITLE
move IsFiveDataModule into lit_utils

### ISSUE THREAD

**Issue by charlesfrye** [wandb/edu#60]
The switch to normalizing the input data with batchnorm means we don't need students to edit the `DataModule` directly, so we can pull that code out -- it's distracting and `DataLoader`s aren't needed for any of the other exercises, here or in other notebooks.
